tfx/components/base/base_executor.py

```diff
@@ -160,3 +160,116 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
          output_dict: Dict[Text, List[types.Artifact]],
          exec_properties: Dict[Text, Any]) -> None:
     pass
+
+
+class BeamExecutor(BaseExecutor):
+  """ Abstract TFX executor class for handling Beam-related logic."""
+
```

Comment:
Can you add a line to the effect "Experimental: no backwards compatibility guarantees."?

Reply:
Done.